app/javascript/src/tests/components/CandidateCard.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import { expect, describe, it, vi, afterEach } from 'vitest'
+import { render, cleanup } from '@testing-library/react'
+import CandidateCard from '@/components/CandidateCard.jsx'
+import createCandidate from '@/models/candidate'
+
+vi.mock('antd', () => ({
+  Card: ({ children }) => <div>{children}</div>,
+  Avatar: ({ children }) => <div>{children}</div>,
+  Flex: ({ children }) => <div>{children}</div>,
+  Typography: {
+    Title: ({ children }) => <div>{children}</div>,
+  },
+}))
+
+describe('CandidateCard', () => {
+  afterEach(cleanup)
```

Comment:
Using cleanup because of vitest issue of not being automatically called upon https://github.com/vitest-dev/vitest/issues/1430